src/main/java/org/broadinstitute/hellbender/tools/examples/ExampleAssemblyRegionWalkerSpark.java

```diff
@@ -77,7 +77,7 @@ public ActivityProfileState isActive(AlignmentContext locusPileup, ReferenceCont
 
     @Override
     protected void processAssemblyRegions(JavaRDD<AssemblyRegionWalkerContext> rdd, JavaSparkContext ctx) {
-        rdd.map(assemblyFunction(knownVariants)).saveAsTextFile(outputFile);
+        rdd.map(assemblyFunction(knownVariants)).coalesce(1).saveAsTextFile(outputFile);
```

Comment:
Good point. I've reverted this since it is tracked by https://github.com/broadinstitute/gatk/issues/2349, and is unrelated to this change.